### ISSUE TITLE
URL Doesn't Exist

### ISSUE THREAD

**Issue by PeteLawler** [adafruit/arduino-board-index#5]
"Adafruit Package Lists for the Arduino v1.6.4+ Board Manager https://adafruit.github.io/arduino-board-index/package_adafruit_index.json"

http://adafruit.github.io gives a 404.

https://raw.githubusercontent.com/adafruit/arduino-board-index/gh-pages/package_adafruit_index.json seems to work fine. I suggest fixing the text here and on the Arduino Unofficial Boards page accordingly

**Comment by ladyada** [comment]
just tried that url and it works - are you sure its down?

**Comment by PeteLawler** [comment]
Very sure it's 404ing for me (though, tbh, I've only had one coffee this morning so there's still a little uncertainty...)

$ telnet adafruit.github.io 80
Trying 103.245.222.133...
Connected to github.map.fastly.net.
Escape character is '^]'.
GET / HTTP/1.0
Host: adafruit.github.io

HTTP/1.1 404 Not Found

**Comment by ladyada** [comment]
but, thats not the url?
you need to use:
https://adafruit.github.io/arduino-board-index/package_adafruit_index.json
that's the url - the full thing...and it works?

**Comment by PeteLawler** [comment]
I'm gunna blame the coffee for this....
Yes, that full URL works.

However, with Privoxy up I get a:
ERR_TUNNEL_CONNECTION_FAILED

So I think what I did before I got the coffee into me was try it out, got the CONNECTION_FAILED, then disabled Privoxy and tested the host and got the 404 leading me to think the thing didn't work at all.

Digging a bit deeper into Privoxy's logs, I see:
2016-04-21 07:56:54.513 7fc7e7fff700 Actions: +block{Host matches generic block pattern.} +change-x-forwarded-for{block} +client-header-tagger{css-requests} +client-header-tagger{image-requests} +hide-from-header{block} +set-image-blocker{pattern} 
2016-04-21 07:56:54.513 7fc7e7fff700 Crunch: Blocked: adafruit.github.io:443
10.2.0.71 - - [21/Apr/2016:07:56:54 +1000] "CONNECT adafruit.github.io:443 HTTP/1.1" 403 9041
2016-04-21 07:56:55.798 7fc800ff9700 Actions: +change-x-forwarded-for{block} +client-header-tagger{css-requests} +client-header-tagger{image-requests} +hide-from-header{block} +set-image-blocker{pattern} 

So it seems a problem between the site and the default Privoxy rules. My humblest apologies for not properly debugging in the first instance.
